dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HttpUtils.java

```diff
@@ -33,66 +33,86 @@
  * http utils
  */
 public class HttpUtils {
-	
-	
-	public static final Logger logger = LoggerFactory.getLogger(HttpUtils.class);
 
-	/**
-	 * get http request content
-	 * @param url url
-	 * @return http get request response content
-	 */
-	public static String get(String url){
-		CloseableHttpClient httpclient = HttpClients.createDefault();
+    public static final Logger logger = LoggerFactory.getLogger(HttpUtils.class);
 
-		HttpGet httpget = new HttpGet(url);
-		/** set timeout、request time、socket timeout */
-		RequestConfig requestConfig = RequestConfig.custom().setConnectTimeout(Constants.HTTP_CONNECT_TIMEOUT)
-				.setConnectionRequestTimeout(Constants.HTTP_CONNECTION_REQUEST_TIMEOUT)
-				.setSocketTimeout(Constants.SOCKET_TIMEOUT)
-				.setRedirectsEnabled(true)
-				.build();
-		httpget.setConfig(requestConfig);
-		String responseContent = null;
-		CloseableHttpResponse response = null;
+    /**
+     * get http request content
+     *
+     * @param url url
+     * @return http get request response content
+     */
+    public static String get(String url) {
+	boolean kerberState = PropertyUtils.getBoolean(Constants.HADOOP_SECURITY_AUTHENTICATION_STARTUP_STATE, false);
```

Comment:
Hi~,
Because KERBER is only used for HADOOP_SECURITY_AUTHENTICATION, which belongs to a specific business logic. Therefore, the judgment of kerberstate suggests that it should be implemented in the business logic class not uilts class, and it is recommended to write a new get method specifically used to return KerberosHttpclient.

For example: If the user sets kerberstate to true, then when we use httputils.get in other places, a new kerberhttpclient will appear instead of a closed ablehttpclient.

-------

因为kerber只用于HADOOP_SECURITY_AUTHENTICATION，其属于一种特定的业务逻辑。所以kerberState的判断建议不要放在utils中，应该放在业务逻辑类中，并且建议再新写一个get方法专门用于返回KerberosHttpClient。

举例：如果用户设置了kerberState为true，那么我们在其他地方用到HttpUtils.get时，也会出现新建一个kerberHttpClient，而不是CloseableHttpClient。

If you have any question or suggestion, please put forward~

Reply:
It has been modified.